### ISSUE TITLE
[a11y][Violation][1.4.3 Contrast (Minimum)] Issues reported on Notebook UI using IBM Equal Access Accessibility Checker tool

### ISSUE THREAD

**Issue by nkn2022** [jupyter/notebook#7490]
## Description

I am running an accessibility test suite with Notebook UI using the **IBM Equal Access Accessibility Checker** tool on Chrome browser. The results look good overall, only a few things are missing to reach a first formal level of compliance. So kindly help look into these. 

Issues reported were violating **WCAG 2.2** requirements. 
Ref: https://www.ibm.com/able/requirements/checker-rule-sets

Full report (includes information on element locations where issue were found): 
[Accessibility_Report-7a73e10e-f8a4-4a06-a78d-5174a1309c5b.xlsx](https://github.com/user-attachments/files/17400350/Accessibility_Report-7a73e10e-f8a4-4a06-a78d-5174a1309c5b.xlsx)

# Category: `1.4.3 Contrast (Minimum)`
Ref: https://www.ibm.com/able/requirements/requirements/?version=v7_3#1_4_3)

****Violation of Rule:** `text_contrast_sufficient`**

> List of issues with elements that violate this rule with a screenshots
> > **Issue 1** 
> > Text contrast of 4.17 with its background is less than the WCAG AA minimum requirements for text of size 13px and weight of 400
> > **Element location:** 
> > `<span class="ͼ11">`
> > **Screenshot:**
> > ![Image](https://github.com/user-attachments/assets/a5460e7e-dfc8-4b1a-8d6a-0cb810815949)

> > **Issue 2** 
> > Text contrast of 4.26 with its background is less than the WCAG AA minimum requirements for text of size 13px and weight of 400
> > **Element location:** 
> > `<span class="ͼu">`
> > **Screenshot:**
> > ![Image](https://github.com/user-attachments/assets/aaaebb87-59b9-4e1b-9888-966a1823269b)

## Reproduce

- Install Chrome extension for IBM accessibility checker tool. Ref: https://www.ibm.com/able/toolkit/verify/automated
- Open Notebook UI on Chrome browser. Add some cells with code.
- Run the tool and we can see the above `Violation`s reported.

Here is the screenshot of the UI scanned for the attached report:
![Image](https://github.com/user-attachments/assets/895eebec-51ad-43cd-93bb-99c4a8193b68)

## Expected behavior

No violations are reported on the scanning to reach a first formal level of compliance.

**Comment by nkn2022** [comment]
@RRosio The JupyterLab [issue](https://github.com/jupyterlab/jupyterlab/issues/16888) has been closed as a false positive for the tool. But please note that the issues mentioned here reference different areas of the UI. So these will need further investigation.

**Comment by nkn2022** [comment]
Tested on Notebook v7.4.7 and still see these issues reported.
